contracts/contracts/strategies/ConvexEthMetaStrategy.sol

```diff
@@ -21,19 +21,23 @@ contract ConvexEthMetaStrategy is InitializableAbstractStrategy {
     using StableMath for uint256;
     using SafeERC20 for IERC20;
 
-    uint256 internal constant MAX_SLIPPAGE = 1e16; // 1%, same as the Curve UI
-    address internal constant ETH_ADDRESS =
+    uint256 public constant MAX_SLIPPAGE = 1e16; // 1%, same as the Curve UI
+    address public constant ETH_ADDRESS =
         0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE;
-    address internal cvxDepositorAddress;
-    IRewardStaking public cvxRewardStaker;
-    uint256 internal cvxDepositorPTokenId;
-    ICurveETHPoolV1 internal curvePool;
-    IERC20 internal lpToken;
-    IERC20 internal oeth;
-    IWETH9 internal weth;
+    address public immutable cvxDepositorAddress;
+    IRewardStaking public immutable cvxRewardStaker;
+    uint256 public immutable cvxDepositorPTokenId;
+    ICurveETHPoolV1 public immutable curvePool;
+    IERC20 public immutable lpToken;
+    IERC20 public immutable oeth;
+    IWETH9 public immutable weth;
```

Comment:
Is this going to change the storage slots, for future upgrades on top of this? Maybe we should just have a depreciated static sized array?